Assets/MixedRealityToolkit/_Core/Utilities/DebugUtilities.cs

```diff
@@ -1,56 +1,168 @@
+        /// object's currently configured logging level.
+        /// </summary>
+        /// <param name="message">The message to display.</param>
+        /// <remarks>
+        /// To help differentiate the output of this method from that of LogError, the
+        /// string "Critical: " is prepended to the supplied message.
+        /// </remarks>
+        public static void LogCriticalError(string message)
+        {
+//            LogCriticalError(MixedRealityToolkitLoggingLevel, message);
+        }
+
+        /// <summary>
+        /// Logs a critial error message.
+        /// </summary>
+        /// <param name="logLevel">The logging level used to filter debug output.</param>
+        /// <param name="message">The message to display.</param>
+        /// <remarks>
+        /// To help differentiate the output of this method from that of LogError, the
+        /// string "Critical: " is prepended to the supplied message.
+        /// </remarks>
+        public static void LogCriticalError(LoggingLevels logLevel, string message)
         {
-            Debug.Assert(condition, message);
+            if ((logLevel & LoggingLevels.CriticalError) != 0)
+            {
+                Debug.LogError($"Critical: {message}");
+            }
         }
 
         /// <summary>
-        /// Asserts a condition.
+        /// Logs an error message, using the Mixed Reality Toolkit
+        /// object's currently configured logging level.
         /// </summary>
-        /// <param name="condition">The condition that is expected to be true.</param>
-        public static void DebugAssert(bool condition)
+        /// <param name="message">The message to display.</param>
+        public static void LogError(string message)
         {
-            DebugAssert(condition, string.Empty);
+//            LogError(MixedRealityToolkitLoggingLevel, message);
         }
 
         /// <summary>
         /// Logs an error message.
         /// </summary>
-        /// <param name="condition">The message to log.</param>
-        public static void DebugLogError(string message)
+        /// <param name="logLevel">The logging level used to filter debug output.</param>
+        /// <param name="message">The message to display.</param>
+        public static void LogError(LoggingLevels logLevel, string message)
         {
-            Debug.LogError(message);
+            if ((logLevel & LoggingLevels.Error) != 0)
+            {
+                Debug.LogError(message);
+            }
         }
 
         /// <summary>
-        /// Logs a warning message.
+        /// Logs an informational message, using the Mixed Reality Toolkit
+        /// object's currently configured logging level.
+        /// </summary>
+        /// <param name="message">The message to display.</param>
+        public static void LogInformation(string message)
+        {
+//            LogInformation(MixedRealityToolkitLoggingLevel, message);
```

Comment:
same question

Reply:
I was testing something and missed uncommenting it... thanks for the catch!